release/conf/nameserver.flags

```diff
@@ -34,3 +34,5 @@
 
 #--replica_num=3
 #--partition_num=8
+--system_table_replica_num=2
```

Comment:
Why 2?

Reply:
gurantee high availability

Reply:
but this flags file is for stand-alone, we just have one tablet server, do 2 replicas make sense?

Reply:
This conf file is not for stand alone only and system table will not be created in standalone mode currently. If necessary, we will support in future

Reply:
ok